### PR TITLE
Fix disfigured simplified form of res.partner

### PR DESCRIPTION
**Pull request by paulius-sladkevicius**
Existing form inherit disfigure form layout.

### CODE REVIEW

**Comment by bwrsandman** [comment]
@komsas could you provide screenshots like I did for https://github.com/OCA/partner-contact/pull/44

It will make review a lot easier

**Comment by paulius-sladkevicius** [comment]
@bwrsandman sorry missed your comment. Do you know If OCA have runbot where we could check it how it looks on 7.0 and how on this branch?

**Comment by bwrsandman** [comment]
@komsas it does, actually:
https://runbot.odoo-community.org/runbot/repo/https-github-com-oca-partner-contact-134

A rebase will probably trigger it.

**Comment by yvaucher** [comment]
@komsas You can do

```
git commit --amend
git push -f
```

to trigger the build of runbot ?

**Comment by paulius-sladkevicius** [comment]
I'm already have CLA it https://odoo-community.org/members/hacbee-uab-paulius-sladkevicius-481, could be an issue because my email changed from @hacbee.com to @hbee.eu

**Comment by gurneyalex** [comment]
CLA ok

**Comment by gurneyalex** [comment]
Runbot warning status unrelated to this PR.

**Comment by gurneyalex** [comment]
I get an error message display the partner form view on runbot:

```
Error: NameError: name 'street' is not defined
```

Probably not related to this PR -> #127

File: partner_firstname/partner_view.xml [hunk 1]
```diff
@@ -8,12 +8,12 @@
         <field name="name" position="attributes">
             <attribute name="attrs">{'readonly': [('is_company', '=', False)], 'required': [('is_company', '=', True)]}</attribute>
         </field>
-        <field name="category_id" position="before">
+        <xpath expr="//div[@class='oe_title']" position="after">
           <group attrs="{'invisible': [('is_company', '=', True)]}">
               <field name="lastname" attrs="{'required': [('is_company', '=', False)]}"/>
             <field name="firstname" />
```
Comment (gurneyalex) [inline partner_firstname/partner_view.xml]: could you fix the indentation here?

**Comment by gurneyalex** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**